app/models/attachable.rb

```diff
@@ -60,6 +67,21 @@ def attachables
     [self]
   end
 
+  def has_attachment_changes?
+    created_attachments.any? || updated_attachments.any? || deleted_attachments.any?
+  end
+
+  def created_attachments
+    attachments.where("created_at > ?", created_at + CASE_CREATION_TIME)
+  end
+
+  def updated_attachments
+    updated_attachments = attachments.where("updated_at > ?", created_at) ||
+      attachments.join(:govspeak_content).where("govspeak_content.updated_at > ?", govspeak_content.created_at)
+
+    updated_attachments - created_attachments
+  end
```

Comment:
I'm going to have a quick think about this one – I have a feeling there's going to be a nicer way to do it 🤔

In particular, I'm not sure the `||` is helpful here. What about if we have one `FileAttachment` which has changed name (`attachments.where("updated_at > ?", created_at)`) and also a `HtmlAttachment` which has some updated Govspeak content? I think we'll only get one OR the other right now.